wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift

```diff
@@ -369,36 +369,73 @@ extension WireCallCenterV3 {
+            zmLog.safePublic("Invalid active speakers data")
+            return
+        }
 
-            // Example of `data`
-            //  {
-            //      "audio_levels": [
-            //          {
-            //              "userid": "3f49da1d-0d52-4696-9ef3-0dd181383e8a",
-            //              "clientid": "24cc758f602fb1f4",
-            //              "audio_level": 100,
-            //              "audio_level_now": 100
-            //          }
-            //      ]
-            // }
+        // Example of `data`
+        //  {
+        //      "audio_levels": [
+        //          {
+        //              "userid": "3f49da1d-0d52-4696-9ef3-0dd181383e8a",
+        //              "clientid": "24cc758f602fb1f4",
+        //              "audio_level": 100,
+        //              "audio_level_now": 100
+        //          }
+        //      ]
+        // }
 
-            do {
-                let change = try self.decoder.decode(AVSActiveSpeakersChange.self, from: data)
-                if let call = self.callSnapshots[conversationId] {
-                    self.callSnapshots[conversationId] = call.updateActiveSpeakers(change.activeSpeakers)
+        do {
+            let change = try self.decoder.decode(AVSActiveSpeakersChange.self, from: data)
+
+            if let call = self.callSnapshots[conversationId] {
+
+                self.callSnapshots[conversationId] = call.updateActiveSpeakers(change.activeSpeakers)
+
+                guard self.isSignificantActiveSpeakersChange(
+                    change: change,
+                    in: call
+                ) else {
+                    return
+                }
+
+                handleEventInContext("active-speakers-change") {
                     WireCallCenterActiveSpeakersNotification().post(in: $0.notificationContext)
                 }
-            } catch {
-                zmLog.safePublic("Cannot decode active speakers change JSON")
             }
+        } catch {
+            zmLog.safePublic("Cannot decode active speakers change JSON")
         }
     }
 
+    private func isSignificantActiveSpeakersChange(
+        change: AVSActiveSpeakersChange,
+        in call: CallSnapshot
+    ) -> Bool {
+        let currentSpeakers = Set(call.activeSpeakers)
+        let newSpeakers = Set(change.activeSpeakers)
+
+        var isSignificant = false
+
+        for newSpeaker in newSpeakers {
+            let currentSpeaker = currentSpeakers.first {
+                $0.client.avsIdentifier == newSpeaker.client.avsIdentifier
+            }
+
+            if let currentSpeaker {
+                let stoppedTalking = currentSpeaker.audioLevelNow > 0 && newSpeaker.audioLevelNow == 0
+                let startedTalking = currentSpeaker.audioLevelNow == 0 && newSpeaker.audioLevelNow > 0
+
+                isSignificant = stoppedTalking || startedTalking
+            } else if newSpeaker.audioLevelNow > 0 {
+                isSignificant = true
+            }
```

Comment:
Maybe a bit simpler?
```suggestion
            } else {
                isSignificant = newSpeaker.audioLevelNow > 0
            }
```

Reply:
applied https://github.com/wireapp/wire-ios/pull/1507/commits/f8dfdca98ceb239bf997d7ff035b4e8bce051cb8